src/Cthulhu.jl

```diff
@@ -212,7 +212,14 @@ end
 function _descend(term::AbstractTerminal, interp::CthulhuInterpreter, mi::MethodInstance;
     override::Union{Nothing,InferenceResult}=nothing, debuginfo::Union{Symbol,DebugInfo}=DInfo.compact, # default is compact debuginfo
     params=current_params(), optimize::Bool=true, interruptexc::Bool=true,
-    iswarn::Bool=false, verbose=true, inline_cost::Bool=false)
+    iswarn::Bool=false, hide_type_stable::Union{Nothing,Bool}=nothing, verbose::Union{Nothing,Bool}=nothing, inline_cost::Bool=false)
+    if !(isnothing(verbose) || isnothing(hide_type_stable))
+        error("`verbose` is being replaced with `hide_type_stable`, do not use both")
+    end
```

Comment:
We probably don't really need to make this an error if we have the deprecation warning.

Reply:
If the user gives both and they conflict, which one wins?

Reply:
The non-deprecated one. But it doesn't really mater, we might as well leave this as-is.